cht-petals/setup-petals.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Usage: setup-petals.sh [--model-path=<DIR>] [--dht-prefix=<PREFIX>] [--port=<INT>]
+set -eEuo pipefail
+
+tmpdir="${PREM_APPDIR:-.}/petals-$(uuid)"
+
+cleanup(){
+  for i in $(jobs -p); do
+    kill -n 9 $i || :
+  done
+  rm -rf "$tmpdir"
+  exit 0
+}
+
+trap "cleanup" SIGTERM
+trap "cleanup" SIGINT
+trap "cleanup" ERR
+
+# clone source
+git clone -n --depth=1 --filter=tree:0 https://github.com/premAI-io/prem-services.git "$tmpdir"
```

Comment:
btw you should change the URL to `https://github.com/biswaroop1547/prem-services.git` when testing this PR locally

Reply:
okay

Reply:
Not needed anymore since it's merged